PRISMRendering/PRISMRenderingLogic/PRISMRenderingLogic.py

```diff
@@ -201,6 +201,12 @@ def setPlacingMarkups(self, paramType, paramName, btn, interaction = 1, persiste
     
     ## Current markup being modified
     self.currentMarkupBtn = btn
+
+    # Getting the "EndPoints" node (always first MarkupsFiducial created)
+    node = slicer.mrmlScene.GetNodeByID("vtkMRMLMarkupsFiducialNode1")

```

Comment:
Is this always the case? If another module creates a fiducial node before, isn't it going to create problems?

Reply:
Ah yeah you're right, if the user wants to use several module and one of these creates a markup, it wont work.
I thought getting the node with the ID would be better then its name, but sounds buggy in that case. I'm gonna change this ASAP

Reply:
Getting the node with the id is a good idea. The id is unique and cannot be changed by users. However, you have to store the id of the node in a variable when you create it and use the id to retrieve the node every time you need it. You should probably check that the node is still valid every time you try to access it. Writing an accessor function that performs those checks for you might be a good idea.